ibis/backends/conftest.py

```diff
@@ -730,3 +731,38 @@ def test_employee_data_2():
     )
 
     return df2
+
+
+class PackageDiscarder:
+    def __init__(self):
+        self.pkgnames = []
+
+    def find_spec(self, fullname, path, target=None):
+        if fullname in self.pkgnames:
+            raise ImportError(fullname)
+
+
+@pytest.fixture
+@pytest.mark.usefixtures("backend")
+def no_duckdb():
```

Comment:
This can be done simpler using `monkeypatch`, and the fact that if `sys.modules["foo"]` returns `None` then an `ImportError` is raised. Something like:

```python
@pytest.fixture
def no_duckdb(monkeypatch):
    monkeypatch.setitem(sys.modules, "duckdb", None)
```

Reply:
that's much nicer, thanks!

Reply:
Very nice indeed, I did no know about that either.